### PR TITLE
VTCCA-4043: fixed issues found in testing

### PR DESCRIPTION
**Pull request by habeeb01**

### CODE REVIEW

File: app/controllers/agentAppointment/AppointAgentController.scala [hunk 1]
```diff
@@ -279,6 +257,33 @@ class AppointAgentController @Inject()(
       )
   }
 
+  private def appointAgentPropertiesBadRequest(errors: Form[_])(implicit request: BasicAuthenticatedRequest[_]) = {
+    val data: Map[String, String] = errors.data
+    accounts.withAgentCode(data("agentCode")).flatMap {
```
Comment (darrell-plant) [inline app/controllers/agentAppointment/AppointAgentController.scala:262]: Is ```data("agentCode")``` guaranteed not to throw NoSuchElementException here?

**Comment by darrell-plant** [comment]
LGTDz 👍

File: app/controllers/agentAppointment/AppointAgentController.scala (hunk 1 above)
Comment (habeeb01) [reply app/controllers/agentAppointment/AppointAgentController.scala:262]: yes, agentCode is a hidden field in both forms on the page

**Comment by reddy-yattapu** [comment]
:+1:

**Pull request merged**